helix-term/src/keymap/default.rs

```diff
@@ -183,6 +183,8 @@ pub fn default() -> HashMap<Mode, KeyTrie> {
         "C-f" | "pagedown" => page_down,
         "C-u" => page_cursor_half_up,
         "C-d" => page_cursor_half_down,
+        "S-left" => hover_previous,
+        "S-right" => hover_next,
```

Comment:
nice work.

I am not a helix expert. I also introduced new commands to cycle through the function signatures. But later I discovered, that it is possible to define `handle_event` in the popup, where you can handle all possible key events. see this [example](https://github.com/helix-editor/helix/blob/39617fceb3302f1dfb6bb73c595551b00773657b/helix-term/src/ui/lsp.rs#L69)

IMO maybe this is a little bit cleaner, since we don't need to define new commands for the same functionality. (another thing is that maybe we should use the same key bindings inside all popups: completion, signature and hover)

As I said I am not a helix expert, so let's wait for maintainer review.

Reply:
yeah these should probably be popup bindings

Reply:
Thanks for pointing that out. I agree this is much clearer. 

I will replace the command keymaps with this approach in the `Hover` component `handle_event`.